aws/lambda/lambda_function.py

```diff
@@ -13,7 +13,7 @@ def handler(event, context):
     # Extract URL from the request
     logging.info("Received event: " + json.dumps(event, indent=2))
     data = json.loads(json.dumps(event))
-    url = data["url"]
+    url = json.loads(data["body"])["url"]
```

Comment:
nit - Similar to inference response, expected request formate would be helpful

Reply:
Added a request example in the comment 👍